MoltenVK/MoltenVK/GPUObjects/MVKImage.mm

```diff
@@ -2308,7 +2308,10 @@ static void signalAndUntrack(const MVKSwapchainSignaler& signaler) {
 // Memory detached in destructor too, as a fail-safe.
 MVKImageView::~MVKImageView() {
 	detachMemory();
-	_image->release();
+    if (_image) {
+        _image->release();
+        _image = NULL;
```

Comment:
It shouldn't be necessary to set the `_image` to `nullptr` here, since the object will cease to exist after the destructor is run.